samples/test/steps/given.ts

```diff
@@ -45,7 +50,10 @@ Given(
 
 Given(
   /^([^/s]+) is a user with a verified email and a set password$/,
-  createAndStoreUserInContext
+  async function(firstName: string) {
```

Comment:
why is this needed? different context (this)? if that's the case, why not just `createAndStoreUserInContext.bind(this)`?

Reply:
I've added 2nd param `assignToGroup?: string` to `createAndStoreUserInContext ` function.
And Cucumber supports only 1 parameter in callback function

Reply:
I see, looks like you can use `curry` for in that case. like `foo(arg1)(arg2)`. But not needed, there is no issue with current implementation.